dev/release/release_rc.sh

```diff
@@ -109,13 +109,13 @@ if [ "${RELEASE_SIGN}" -gt 0 ]; then
     if [ -n "${run_id}" ]; then
       break
     fi
-    sleep 60
+    sleep 600
```

Comment:
We have to wait 10 minutes!?

Reply:
As it's a `sleep`, it seems so.  It's maybe related to @kou network. 10mn is OK though is necessary.

Reply:
Ah the issue explains the whole process takes ~30 minutes so ~10 minutes seems reasonable (I guess Github doesn't like pinging too often)

Reply:
Yes, it's pretty clear regarding https://github.com/apache/arrow-java/issues/588#issue-2832065440

Reply:
Yes.
I've added it to this PR description too. Sorry. I should have explained it in the PR description too.